server/game/PlayActions/MarshalCardAction.js

```diff
@@ -42,7 +42,7 @@ class MarshalCardAction extends BaseAbility {
         };
         context.game.raiseEvent('onCardMarshalled', params, () => {
             context.player.putIntoPlay(context.source, 'marshal');
-            context.game.addMessage(this.getMessageFormat(params), context.player, context.source, params.originalController, params.originalLocation, params.originalParent, context.costs.gold);
+            context.game.queueSimpleStep(() => context.game.addMessage(this.getMessageFormat(params), context.player, context.source, params.originalController, params.originalLocation, params.originalParent, context.costs.gold));
```

Comment:
What's the purpose of delaying these messages (along with the other PlayActions) by wrapping them in `queueSimpleStep`? It will make the game log non-linear if there are interrupts (and potentially even reactions). e.g.

With queue:
```
Player 2 plays X (interrupt to player 1 marshalling Y)
Player 1 marshals Y
```

Without queue:
```
Player 1 marshals Y
Player 2 plays X (interrupt to player 1 marshalling Y)
```

Reply:
I had definitely done this for a reason, at least initially (whilst implementing all cards for the reveal update), but I'll need to re-investigate why this was the case. Could be legitimate, could also just be left-over changes I forgot to revert.

Reply:
@ystros Honestly couldn't find the proper reason for this adjustment, and reverting it does not fail any tests. I'll revert it, and if we find any issues in the future, we can deal with it then.

Reverted in https://github.com/throneteki-playtesting/throneteki/commit/143c98f861c602e4ac998bbcf9c06a0b9ad81211

Can mark this as resolved! :)